packages/Python/lldbsuite/test/lang/swift/expression/class_constrained_protocol/TestClassConstrainedProtocol.py

```diff
@@ -23,7 +23,6 @@ def test_extension_weak_self (self):
         self.build()
         self.do_self_test("Break here for weak self")
 
-    @expectedFailureAll(bugnumber="rdar://31822722")
```

Comment:
This needs to be marked as swift-test (not your fault, but while you're around, please fix it).